Lib/glyphsLib/builder/__init__.py

```diff
@@ -148,12 +151,49 @@ def to_designspace(
     return builder.designspace
 
 
-def preflight_glyphs(font):
+def preflight_glyphs(font, *, glyph_data=None, **todos):
     """Run a set of transformations over a GSFont object to make
-    it easier to convert to UFO; resolve all the "smart stuff"."""
+    it easier to convert to UFO; resolve all the "smart stuff".
+
+    Currently, the transformations are:
+        - `propagate_all_anchors`: copy anchors from components to their parent
+
+    More transformations may be added in the future.
+
+    Some transformations may have custom parameters that can be set in the
+    font. For example, the `propagate_all_anchors` transformation can be
+    disabled by setting the custom parameter "Propagate Anchors" to False
+    (see `TRANSFORMATION_CUSTOM_PARAMS`).
+
+    Args:
+        font: a GSFont object
+        glyph_data: an optional GlyphData object associating various properties to
+            glyph names (e.g. category) that overrides the default one
+        **todos: a set of boolean flags to enable/disable specific transformations,
```

Comment:
I'm a bit confused about the name; shouldn't they be called something like `flags`?